docker-images/test-client/scripts/consumer_config_generator.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Write the config file
+cat <<EOF
+# Provided configuration
+${PRODUCER_CONFIGURATION}
+
+# Fixed configuration
+ssl.keystore.password=${CERTS_STORE_PASSWORD}
+ssl.truststore.password=${CERTS_STORE_PASSWORD}
+ssl.endpoint.identification.algorithm=
```

Comment:
Why do we have this hardcoded as disabled?

Reply:
Because the last paragraph in http://strimzi.io/docs/master/#kafka_client_connections_via_tls says I need this. I admit I didn't try without it.

Reply:
Hmm, we should have SAN support now - I think we should update it. Not sure what is the current state of this docu. :-(